app/javascript/components/patient/Patient.js

```diff
@@ -20,6 +21,7 @@ class Patient extends React.Component {
       expandArrivalNotes: false,
       expandPlannedTravelNotes: false,
       primaryLanguageDisplayName: null,
+      showUpdateFlagModal: false,
```

Comment:
one quick thing, this var name is different than the one used for toggling below, so this just needs to be changed

Reply:
Great catch, will include this fix in my next commit!